homeassistant/components/mqtt/fan.py

```diff
@@ -172,7 +174,15 @@ def valid_preset_mode_configuration(config):
     }
 ).extend(MQTT_ENTITY_COMMON_SCHEMA.schema)
 
+# The use of PLATFORM_SCHEMA is deprecated in HA Core 2022.6
```

Comment:
```suggestion
# Configuring MQTT Fans under the fan platform key is deprecated in HA Core 2022.6
```

Reply:
We might need to update that in call coming PR's as well.